Calculator/Services/NumberService.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Calculator.Services
+{
+    public class NumberService : INumberService
+    {
+        private string[] DefaultDelimiters = { ",", "\n" };
+        private string[] DelimiterSurroundingStrings = { "//", "[", "][", "]" };
```

Comment:
You're mixing naming standards - this is the standard for constants, which these are not.